src/boilerplate/content.js

```diff
@@ -11,7 +11,7 @@ export default {
 
     async eventListeners() {
         window.addEventListener('message', (event) => {
-            this._gameContent = event.data;
+            this._gameContent = JSON.parse(event.data);
```

Comment:
is it worth moving this to base template?

Reply:
also, `message` as event name is too generic and can be renamed to `contentData` or `styleData`

Reply:
it has to be message, that's the event type